source/server/http/admin.h

```diff
@@ -319,11 +320,15 @@ class AdminImpl : public Admin,
     Stats::Scope& listenerScope() override { return *scope_; }
     uint64_t listenerTag() const override { return 0; }
     const std::string& name() const override { return name_; }
+    Server::ActiveUdpListenerFactory* udpListenerFactory() override {
+      return udp_listener_factory_.get();
```

Comment:
Can't you just return nullptr here or better NOT_IMPLEMENTED?

Reply:
I think it's hard coded to TCP currently: https://github.com/envoyproxy/envoy/blob/15d54866d6a627accb2ca42a23367c66a83aaf18/source/server/http/admin.cc#L1177 which I think is the only thing that makes sense.

Optimally, you would modify admin to fail if someone configures UDP? But either way this code can't get hit so I think it can just be not-implemented.

/wait

Reply:
done